devo/sender/data.py

```diff
@@ -470,44 +483,13 @@ def from_config(config, con_type=None, logger=None):
         :param logger: logger handler, default None
         :return: Sender object
         """
-        con_type = config['type'].upper() if "type" in config and con_type \
-                                             is not None else "SSL"
-        cert_reqs = config['cert_reqs'] if "cert_reqs" in config else True
+        if "cert_reqs" not in config.keys():
```

Comment:
para asignar un booleano dependiendo de una condiciones con un if es más limpio esto

`config['cert_reqs'] = "cert_reqs" not in config.keys()`

Reply:
No estoy asignando un booleano, estoy forzándolo a True si el usuario no ha puesto esa key, pero puede existir de base estando a False o a True

Si hiciera eso me metería false en config['cert_reqs'] si el usuario ha puesto esa key a "True", por ejemplo. Si la ha puesto a false daría igual, pero el problema es que te rompe el true

Reply:
cierto 👍